spec/factories/amusement_parks.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :amusement_park do
```

Comment:
While this is great practice, with such a limited time to complete these challenges, I recommend not spending time with this for these.